Tribler/Test/test_as_server.py

```diff
@@ -518,6 +520,9 @@ def startTest(self, callback, min_callback_delay=5, autoload_discovery=True,
         # Always start testing from the same dir (repo root)
         os.chdir(os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..')))
 
+        # Override default HTTP API enabled to disable API in tests
+        sessdefaults['http_api']['enabled'] = False
```

Comment:
Unfortunately, I cannot manipulate the `SessionConfig` object that is used when the GUI is started in tests. That's why I'm overwriting the defaults instead.